lms/djangoapps/courseware/plugins.py

```diff
@@ -183,9 +183,9 @@ class ProctoringCourseApp(CourseApp):
     @classmethod
     def is_available(cls, course_key: CourseKey) -> bool:
         """
-        Proctoring is available for all courses.
+        Returns true if the proctoring app is available for all courses.
         """
-        return settings.FEATURES.get("ENABLE_SPECIAL_EXAMS", False)
+        return settings.FEATURES.get('ENABLE_PROCTORED_EXAMS')
```

Comment:
I think it should return default "False"?

Reply:
if the `key` doesn't exist, it would default to `None` automatically, as that's also a falsy value, I think its ok to let default to "default".